### PR TITLE
(Chinese) localize wikis

### PR DESCRIPTION
**Pull request by CoolPlayLin**

### CODE REVIEW

**Review by acheong08 — APPROVED** [review]

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -11,29 +11,44 @@ name: Upload Python Package
...
+        run: make ci
 
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
```
Comment (acheong08) [inline .github/workflows/python-publish.yml:44]: Pending review

**Comment by CoolPlayLin** [comment]
wait for a while, I am localzing the wiki

**Comment by CoolPlayLin** [comment]
@acheong08 I finish the task

File: .vscode/settings.json [hunk 2]
```diff
@@ -6,11 +6,16 @@
...
+    "kwargs",
     "mainloop",
     "padx",
     "pady",
     "tkinter"
   ],
   "python.formatting.provider": "black",
+  "yaml.schemas": {
+    "https://json.schemastore.org/github-workflow.json": "file:///c%3A/Users/CoolPlayLin/Project/ChatGPT/.github/workflows/python-publish.yml"
+  },
```
Comment (acheong08) [inline .vscode/settings.json:20]: ?

**Review by acheong08 — CHANGES_REQUESTED** [review]

**Review by acheong08 — APPROVED** [review]

**Pull request merged**